packages/fast-components-react-msft/src/text-action/examples.data.tsx

```diff
@@ -15,7 +16,7 @@ const examples: ComponentFactoryExample<TextActionProps> = {
     detailData: {
         placeholder: "Placeholder",
         button: {
-            id: "button",
+            id: buttonSchema.id,
```

Comment:
@janechu I'm having an issue with the plugin here; I've added the plugin for button in text action, but I'm finding that the children for the button is throwing an invariant violation. Additionally, even if I change this to a different render prop, it doesn't render as expected. Would love some help figuring this out. As it is - this is blocking this work from going in for v4.0

Reply:
There is a PR up to resolve this issue see https://github.com/Microsoft/fast-dna/pull/1464